src/lib.js

```diff
@@ -1147,8 +1147,8 @@ export const getItems = async (
       }
     }
 
-    if (id in constants.ACCESSORY_UPGRADES) {
-      const accessoryUpgrades = constants.ACCESSORY_UPGRADES[id];
+    if (constants.getUpgradeList(id) !== undefined) {
```

Comment:
`!== undefined` is redundant (used several places). Unless you want to exclude other falsy values, but I don't think you do

Reply:
> `!== undefined` is redundant (used several places). Unless you want to exclude other falsy values, but I don't think you do

Just realized it returns boolean, I'm dumb ;-;